### ISSUE TITLE
Cannot read properties of undefined (reading 'bold') when Nextjs run with Turbopack

### ISSUE THREAD

**Issue by mohammadhosseinmoradi** [denoland/deno#26584]
Version: Deno 2.0.3

I get it error when run with deno.

```
TypeError: Cannot read properties of undefined (reading 'bold')
    at getDefs (file:///D:/Me/Projects/dev/deponet-store/node_modules/.deno/next@15.0.1/node_modules/next/dist/compiled/babel/bundle.js:1848:5496)
    at highlight (file:///D:/Me/Projects/dev/deponet-store/node_modules/.deno/next@15.0.1/node_modules/next/dist/compiled/babel/bundle.js:1848:6631)
    at codeFrameColumns (file:///D:/Me/Projects/dev/deponet-store/node_modules/.deno/next@15.0.1/node_modules/next/dist/compiled/babel/bundle.js:1:77678)
    at formatIssue (file:///D:/Me/Projects/dev/deponet-store/node_modules/.deno/next@15.0.1/node_modules/next/dist/server/dev/turbopack-utils.js:247:20)
    at addErrors (file:///D:/Me/Projects/dev/deponet-store/node_modules/.deno/next@15.0.1/node_modules/next/dist/server/dev/hot-reloader-turbopack.js:810:85)
    at handleProjectUpdates (file:///D:/Me/Projects/dev/deponet-store/node_modules/.deno/next@15.0.1/node_modules/next/dist/server/dev/hot-reloader-turbopack.js:819:25)
detached processes are not currently supported on Windows
```

**Comment by marvinhagemeister** [comment]
What are the steps to get this error? I tried these ones and weren't able to reproduce the error:

1. Create a new next app with `npx create-next-app@latest`
   1.  Pick the `turbopack` option
2. Run `deno task dev` and browse to the site
3. Run `deno task build`

I'm not getting any errors with these steps. Maybe I'm doing something different?

**Comment by marvinhagemeister** [comment]
Came across this other issue, which seems related:

- https://github.com/denoland/deno/issues/25359

**Comment by mohammadhosseinmoradi** [comment]
@marvinhagemeister
I was able to reproduce this issue. When Next.js runs with Node, Next.js displays a dialog describing the error and showing the stack trace, but this doesn’t happen when run with Deno.
[https://github.com/mohammadhosseinmoradi/deno-bug-reproduce-01.git](url)

**Comment by gianpaj** [comment]
the same happens to me
<img width="811" alt="image" src="https://github.com/user-attachments/assets/226d42e7-76a1-4f04-9dfd-f9bffe036b92">

I tried running on this simple Next.js 14.2 repo
https://github.com/gianpaj/custom-cover-letter

**Comment by nathanwhit** [comment]
Thanks, that does reproduce the issue for me!

It looks like the issue is that next with turbopack is using a version of babel that includes an old version of `chalk` which relies on the `__proto__` property. Deno doesn't support `__proto__` by default, for [security reasons](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/proto).

TLDR;

You can add
```
"unstable": ["unsafe-proto"]
``` 
to your `deno.json` and that should resolve the issue. 

That won't be required once a new version of babel with the fixed chalk is released.

**Comment by nicolo-ribaudo** [comment]
> That won't be required once a new version of babel with the fixed chalk is released.

Babel already doesn't depend on chalk anymore. Unfortunately Vercel likes their users to use legacy software, so they always make sure to _bundle_ older versions of Babel and thus simply fixing this in a new Babel version is not enough.

**Comment by nathanwhit** [comment]
> > That won't be required once a new version of babel with the fixed chalk is released.
> 
> Babel already doesn't depend on chalk anymore, and it has not being using chalk in that code path showee in the stack trace since march. Unfortunately Vercel likes their users to use legacy software, so they always make sure to _bundle_ older versions of Babel and thus simply fixing this in a new Babel version is not enough.

Ah thanks for pointing that out! I didn't realize there were 7.x versions without the problematic chalk version. Updated my comment.

**Comment by Flo2410** [comment]
I'm having the same problem with Next.js. Is there a workaround, or do I have to wait until the old babel version gets removed from Next?

**Comment by mintydev789** [comment]
I used to get this error too, but as of Deno version 2.3.3 it works for me, albeit only with `--unstable-unsafe-proto`.
